src/test/java/org/apache/commons/collections4/queue/CircularFifoQueueTest.java

```diff
@@ -421,6 +421,54 @@ public void testGetIndex() {
             assertEquals(confirmed.get(i + 2), queue.get(i));
         }
     }
+	
+	public void testAddNull() {
+        final CircularFifoQueue<E> b = new CircularFifoQueue<>(2);
+        try {
+            b.add(null);
+            fail();
+        } catch (final NullPointerException ex) {
+            assertEquals("Attempted to add null object to queue",ex.getMessage());
```

Comment:
-1: I do not think we should test the actual text of the message, it is too brittle IMO. We should not break test when we are improving or updating exception messages later on. Testing that the exception message is not empty is OK though. Also there is a space missing after the comma.